projects/components/src/radio/radio-group.component.ts

```diff
@@ -40,6 +40,7 @@ import { RadioOption } from './radio-option';
         [ngClass]="[this.optionsDirection, this.disabled || option.disabled ? 'disabled' : '']"
         [value]="option.value"
         [disabled]="option.disabled"
+        [htTooltip]="option.tooltip"
```

Comment:
is this an input parameter or a directive? if is an input name should be tooltip instead of htTooltip.

Reply:
It's the ht tooltip directive.